cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/etl/batch/BatchETLDBTest.java

```diff
@@ -345,29 +354,43 @@ public void testUserNamePasswordCombinations() throws Exception {
 
     Map<String, String> baseSourceProps = ImmutableMap.of(
       Properties.DB.CONNECTION_STRING, hsqlDBServer.getConnectionUrl(),
+      Properties.DB.TABLE_NAME, "my_table",
       Properties.DB.JDBC_PLUGIN_NAME, "hypersql",
       Properties.DB.IMPORT_QUERY, importQuery,
       Properties.DB.COUNT_QUERY, countQuery);
 
     Map<String, String> baseSinkProps = ImmutableMap.of(
       Properties.DB.CONNECTION_STRING, hsqlDBServer.getConnectionUrl(),
+      Properties.DB.TABLE_NAME, "my_table",
       Properties.DB.JDBC_PLUGIN_NAME, "hypersql",
-      Properties.DB.TABLE_NAME, "table",
-      Properties.DB.COLUMNS, "*");
+      Properties.DB.COLUMNS, "*",
```

Comment:
So this was supported until 3.2.0, but I think its broken in 3.2.1. Reported https://issues.cask.co/browse/CDAP-3942. Will handle it separately, because it would have to be fixed in release/3.2.